cmd/generic-handlers.go

```diff
@@ -581,7 +581,18 @@ func hasBadPathComponent(path string) bool {
 	return false
 }
 
-func (h pathValidityHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+// Check if client is sending a malicious request.
```

Comment:
Maybe we do not need to attribute to malice, what is more likely to be a bug.

Reply:
More likely to be a malice. The comment is for the code-reader not to remove this code thinking it is not necessary/important.